src/assert.bash

```diff
@@ -185,12 +185,14 @@ assert_failure() {
 assert_output() {
   local -i is_mode_partial=0
   local -i is_mode_regexp=0
+  local -i use_stdin=0
 
   # Handle options.
   while (( $# > 0 )); do
     case "$1" in
       -p|--partial) is_mode_partial=1; shift ;;
       -e|--regexp) is_mode_regexp=1; shift ;;
+      -) use_stdin=1; shift ;;
```

Comment:
I'm thinking a long option, e.g. `--stdin`, would be nice. Safe for `refute_output`. What do you think?